packages/react-components/theme-designer/src/components/Nav/Nav.tsx

```diff
@@ -1,9 +1,22 @@
 import * as React from 'react';
+import { makeStyles, mergeClasses } from '@griffel/react';
 
 export interface NavProps {
-  className?: String;
+  className: string;
 }
 
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'left',
+    borderBottomWidth: '1px',
+    borderBottomColor: '#D1D1D1',
+    borderBottomStyle: 'solid',
```

Comment:
I forgot to mention this on the parent container. but you can use shorthands to simplify this
```suggestion
   ...shorthands.borderBottom('1px', 'solid', '#D1D1D1')
```